spec/models/efile_submission_spec.rb

```diff
@@ -706,4 +706,17 @@
       end
     end
   end
+
```

Comment:
Adding this spec since I moved `STATE_INTAKE_CLASS_NAMES` form the `efile_submission` model, and I wanted to make sure I didn't break anything.